extras/python-sambacc.spec

```diff
@@ -46,6 +46,13 @@ Recommends: %{name}+validation
 Recommends: %{name}+yaml
 Recommends: %{name}+rados
 %endif
+%if 0%{?rhel} >= 9
+# Enable extras other than validation as the dependency needed
+# is too old on centos/rhel 9.
+Recommends: %{name}+toml
+Recommends: %{name}+yaml
+Recommends: %{name}+rados
+%endif
```

Comment:
In an attempt to avoid repetition we could also consider the following change:

```diff --git a/extras/python-sambacc.spec b/extras/python-sambacc.spec
index 6045131..0509b7d 100644
--- a/extras/python-sambacc.spec
+++ b/extras/python-sambacc.spec
@@ -40,12 +40,14 @@ Summary: %{summary}
 # Distro requires that are technically optional for the lib
 Requires: python3-samba
 Requires: python3-pyxattr
-%if 0%{?fedora} >= 37
+%if 0%{?fedora} >= 37 || 0%{?rhel} >= 9
 Recommends: %{name}+toml
-Recommends: %{name}+validation
 Recommends: %{name}+yaml
 Recommends: %{name}+rados
 %endif
+%if 0%{?fedora} >= 37
+Recommends: %{name}+validation
+%endif
```

Reply:
Sure. Done.